apps/sms/services/js/message_manager.js

```diff
@@ -137,28 +187,39 @@ var MessageManager = {
     };
   },
 
+  /**
+   * Function to get the message object having a given id.
```

Comment:
I'd say we always should add ```@return``` if method returns anything :) In this case I'd say it will be:
```
@returns {DOMRequest.<SmsMessage|MmsMessage>}
```

See [1] http://mxr.mozilla.org/mozilla-central/source/dom/webidl/SmsMessage.webidl . You can also use MXR as good source of webidl that defines various object types.